relay/pkg/btc/remote/remote_test.go

```diff
@@ -1,14 +1,27 @@
 package remote
 
-import "testing"
+import (
+	"context"
+	"testing"
 
-func TestConnect(t *testing.T) {
-	btcChain, err := Connect()
-	if err != nil {
-		t.Fatal(err)
+	"github.com/keep-network/tbtc/relay/pkg/btc"
+)
+
+func TestConnectionWrongURL(t *testing.T) {
+	config := &btc.Config{
+		URL:      "dummyURL",
+		Password: "pass",
+		Username: "user",
+	}
+
+	ctx := context.Background()
+
+	btcChain, err := Connect(ctx, config)
+	if err == nil {
+		t.Fatal("No error received")
 	}
 
-	if btcChain == nil {
-		t.Errorf("btc chain handle is null")
+	if btcChain != nil {
+		t.Errorf("Non nil btc chain received")
 	}
 }
```

Comment:
I think we can remove that test. We're not testing any logic here so we can cover this with an integration test at the end.